Tribes/src/app/app.component.html

```diff
@@ -1,3 +1 @@
-<app-header></app-header>
```

Comment:
why deleted this? we dont want to show the header in every view?

Reply:
no, the login and registration pages will not contain the header

Reply:
ok, but why?

Reply:
design decision :D 
You don't need a login or kingdom/setting/etc header while you'r logging in :)

Reply:
I can see both ways are good.